src/App.jsx

```diff
@@ -1,9 +1,92 @@
 import './App.css';
+import {bestSellingTv, inventory} from "./constants/inventory.js";
+import sellAmount from "./helpers/sellAmount.js";
+import buyAmount from "./helpers/buyAmount.js";
+import currentStockTotal from "./helpers/currentStockTotal.js";
+import productStringGenerator from "./helpers/productStringGenerator.js";
+import priceString from "./helpers/priceString.js";
+import tvScreenSizeString from "./helpers/tvScreenSizeString.js";
+import optionsCheckOrMinus from "./helpers/optionsCheckOrMinus.js";
+import checkIcon from './assets/check.png';
+import notIcon from './assets/minus.png';
 
 function App() {
-  return (
-    <h1>Begin hier met met maken van de applicatie!</h1>
-  )
+
+    function handleClick(buttonText) {
+        console.log(buttonText)
+    }
+
+    return (
+        <div>
```

Comment:
In plaats van een div kun je ook een fragment toevoegen als je geen className invoert.

Reply:
Verwerkt! Goede tip.